cmd/agent/app/integrations.go

```diff
@@ -516,14 +516,14 @@ func parseWheelPackageName(wheelPath string) (string, error) {
 			for scanner.Scan() {
 				line := scanner.Text()
 
-				exp, err := regexp.Compile(wheelPackageName)
+				exp, err := regexp.Compile(wheelPackageNamePattern)
 				if err != nil {
 					return "", fmt.Errorf("regexp internal error: %v", err)
 				}
 
 				matches := exp.FindStringSubmatch(line)
 				if matches == nil {
-					return "", fmt.Errorf("could not find a package name for %s", wheelPath)
+					continue
```

Comment:
Why `continue` here?

Reply:
We are looking for a line that matches this pattern "Name: (\\S+)", we `continue` until we reach the last line.

Previously, it would only work if the `Name: XXX` is at the beginning. Got a case where `Name: XXX` is not on the first line.

Since the file is not large, another possible implementation would be matching the whole file at once. That might simplify the logic by avoid one for loop.